pkg/apis/kops/instancegroup.go

```diff
@@ -59,12 +58,15 @@ const (
 	InstanceGroupRoleNode InstanceGroupRole = "Node"
 	// InstanceGroupRoleBastion is a bastion role
 	InstanceGroupRoleBastion InstanceGroupRole = "Bastion"
+	// InstanceGroupRoleAPIServer is an API server role
+	InstanceGroupRoleAPIServer InstanceGroupRole = "Apiserver"
```

Comment:
Consider capitalizing the new one differently:
```suggestion
	InstanceGroupRoleAPIServer InstanceGroupRole = "APIServer"
```